test/blackbox-tests/test-cases/merlin/github4125.t/dune-workspace

```diff
@@ -0,0 +1,9 @@
+(lang dune 2.0)
+
+(context (default))
+
+(context
+ (opam
+  (name cross)
+  (switch default)
+  (merlin)))
```

Comment:
Sorry for that, I thought other tests were already relying on such a switch but they must have been disabled since.
@rgrinberg said it might be possible to test this kind of feature but I did not found examples of such things in the test-base.

I think disabling this test is fine if don't find a workaround.